### ISSUE TITLE
iPython notebook not loading even with Jupyter extension

### ISSUE THREAD

**Issue by ad12** [microsoft/vscode#126513]
Issue Type: <b>Bug</b>

1. Connect to remote server
2. Install Pylance
3. Create sample.ipynb

This results in a blank screen, with no jupyter notebook interface

VS Code version: Code 1.57.0 (b4c1bd0a9b03c749ea011b06c6d2676c8091a70c, 2021-06-09T17:22:31.215Z)
OS version: Darwin x64 20.4.0
Restricted Mode: No
Remote OS version: Linux x64 5.8.0-53-generic
Remote OS version: Linux x64 5.8.0-53-generic
Remote OS version: Linux x64 5.4.89+

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz (12 x 2900)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|5, 4, 3|
|Memory (System)|32.00GB (1.38GB free)|
|Process Argv|--crash-reporter-id 1572910a-3d68-41d5-8b2d-a8224b00f95b|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: torino|
|OS|Linux x64 5.8.0-53-generic|
|CPUs|Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz (32 x 1205)|
|Memory (System)|125.80GB (0.72GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: torino|
|OS|Linux x64 5.8.0-53-generic|
|CPUs|Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz (32 x 1205)|
|Memory (System)|125.80GB (0.72GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|pod-arjun in pod-arjun (gcr.io/hai-gcp-fine-grained/default)|
|OS|Linux x64 5.4.89+|
|CPUs|Intel(R) Xeon(R) CPU @ 2.20GHz (8 x 2199)|
|Memory (System)|29.40GB (13.44GB free)|
|VM|0%|
</details><details><summary>Extensions (8)</summary>

Extension|Author (truncated)|Version
---|---|---
remote-containers|ms-|0.183.0
remote-ssh|ms-|0.65.7
remote-ssh-edit|ms-|0.65.7
remote-wsl|ms-|0.56.5
vscode-remote-extensionpack|ms-|0.21.0
python|ms-|2021.5.926500501
vscode-pylance|ms-|2021.6.1
jupyter|ms-|2021.6.999406279

</details>

**Comment by rebornix** [comment]
@ad12 can you open Developer Tools and share logs from there?

**Comment by ad12** [comment]
@rebornix which particular logs would be helpful? this is the console output:

![image](https://user-images.githubusercontent.com/12801331/122277796-21f6b080-ce9b-11eb-85ad-5e6a28d39524.png)

**Comment by rebornix** [comment]
@ad12 the logs didn't show any errors which can relate to this. Can you check if you have `workbench.editorAssociations` in your user setting and share the value?

**Comment by ad12** [comment]
@rebornix Yup, I do have it, and it appears to include the correct item,value pair for jupyter notebooks

![image](https://user-images.githubusercontent.com/12801331/122479710-614afd00-cf80-11eb-800b-5bb0ea3d9b93.png)

**Comment by Kimtaehung123** [comment]
same problem

**Comment by seyuboglu** [comment]
Also running into this

**Comment by krandiash** [comment]
I've had this issue too
